.eslintrc.json

```diff
@@ -64,7 +64,18 @@
       // Prevent using optional-chaining in source files, as it is not supported by Polymer analyzer
```

Comment:
Let's move this comment to the respective rule, now that there are multiple of them.

Reply:
Thanks, done.